lib/modules/manager/github-actions/readme.md

```diff
@@ -15,8 +15,10 @@ jobs:
       - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
 ```
 
-Renovate will update the commit SHA but follow the GitHub tag you specified.
+Renovate will update the commit SHA according to the GitHub tag you specified.
 Renovate can update digests that use SHA1 and SHA256 algorithms.
+The GitHub tag is in the format of `(prefix-)(v)1.0.0`, where `prefix` and `v` are optional and `1.0.0` is the version number.
```

Comment:
`v1` and `1.1` are also valid tags and are already captured

Reply:
fixed